node-setup/080_register_stakepool.md

```diff
@@ -30,25 +30,62 @@ At this moment we have:
 
 Registering your stake pool requires:
 
+* Create JSON file with your metadata and store it in the node and in a url you maintain.
+* Get the hash of your JSON file
 * Generate the stake pool registration certificate
 * Create a delegation certificate (pledge)
 * Submit the certificates to the blockchain
 
 **WARNING:** Generating the __stake pool registration certificate__ and the __delegation certificate__ requires the __cold keys__ So, you may want to generate these certificates in your local machine taking the proper security measures to avoid exposing your cold keys to the internet.  
 
-### 1. Generate Stake pool registration certificate
+### 1. Create a JSON file with your pool's metadata
+
+    {
+    "name": "TestPool",
+    "description": "The pool that tests all the pools",
+    "ticker": "TEST",
+    "homepage": "https://teststakepool.com"
+    }
+
+Store the file in your node and in a url you maintain, for example [https://gist.githubusercontent.com/testPool/.../testPool.json](shorturl.at/gDV47)
+
+### 2. Get the hash of your file:
+
+    cardano-cli shelley stake-pool metadata-hash --pool-metadata-file testPool.json
+
+    >6bf124f217d0e5a0a8adb1dbd8540e1334280d49ab861127868339f43b3948af
+    
+### Temporary step until DB-sync is updgraded
+
+Submit a PR to add your pool data to https://github.com/input-output-hk/cardano-ops/blob/master/topologies/ff-peers.nix 
+You will need to provide your IP address/DNS host name and port.
```

Comment:
Perhaps note that the relays can also be registered on-chain, and that can support multiple IPs/DNS names. So this relay registering scheme is also temporary.